deployment/unraid/docker-templates/README.md

```diff
@@ -1,15 +0,0 @@
-# docker-templates
```

Comment:
I don't think we should delete this one, it's used by others outside of our current build process.

Reply:
Restoring.